openpype/hosts/nuke/plugins/publish/collect_writes.py

```diff
@@ -68,12 +68,13 @@ def _set_existing_files_data(self, instance, colorspace):
         representation = self._get_existing_frames_representation(
             instance, collected_frames
         )
-
-        # inject colorspace data
-        self.set_representation_colorspace(
-            representation, instance.context,
-            colorspace=colorspace
-        )
+        files = representation["files"]
+        if isinstance(files, list) and files:
```

Comment:
Why does it **have to be a list** to set colorspaces? I suspect this will now fail to set colorspaces for publishes of a single frame?